x/oracle/keeper/oracle.go

```diff
@@ -101,7 +102,8 @@ func (k Keeper) VoteOracleRegistration(ctx sdk.Context, vote *types.OracleRegist
 		return sdkerrors.Wrap(types.ErrOracleRegistrationVote, err.Error())
 	}
 
-	if k.verifyVoteSignature(ctx, vote, signature) {
+	ok, err := k.verifyVoteSignature(ctx, vote, signature)
+	if err != nil || !ok {
```

Comment:
Later, when implementing the slashing function, this if statement will have to be implemented in a different way.
If err is not nil, slashing should not be performed because it is an error caused by a cause other than signature verification failure.
We probably only need to slash if no error occurs and the ok value is false.

Of course, you don't have to consider it now.

Reply:
Make senses. I will fix it now for safety. I think we can throw a panic if base64 decoding is failed, based on the rule that all values in the `Param` are valid if the `Param` struct was populated successfully.

Reply:
I agree. :)

Reply:
Done: 212d423b02f60219d858f0150efc756a4e021414